docs/dev/compliance.md

```diff
@@ -0,0 +1,120 @@
+# About `inspec compliance`
+
+## Purpose
+
+The `compliance` set of subcommands handle user-initiated communication with Chef Automate. The commands are provided so that a user can interact with an Automate installation.
+
+`inspec compliance` is somewhat analogous to `knife` in that it can be used to upload, download and manage profiles for distribution to other clients.
```

Comment:
```suggestion
`inspec compliance` is somewhat analogous to `knife` in that it can be used to upload, download, and manage profiles for distribution to other clients.
```